plugins/modules/qubesos.py

```diff
@@ -424,7 +424,9 @@ def create(
             )
             vm.netvm = network_vm
         elif vmtype in ["StandaloneVM", "TemplateVM"] and template_vm:
-            vm = self.app.clone_vm(template_vm, vmname, vmtype)
+            vm = self.app.clone_vm(
+                template_vm, vmname, vmtype, ignore_devices=True
```

Comment:
This change technically is unrelated to volumes, but it's desired change so lets keep that in...